openff/toolkit/utils/utils.py

```diff
@@ -436,6 +436,18 @@ def _(obj):
     return unit.Quantity(obj)
 
 
+try:
+    import openmm.unit
+    from openff.units.openmm import from_openmm
+
+    @object_to_quantity.register(openmm.unit.Quantity)
```

Comment:
This is one way to make `mypy` happy here
```suggestion
    from openmm import unit as openmm_unit
    from openff.units.openmm import from_openmm

    @object_to_quantity.register(openmm_unit.Quantity)
```